app/poros/station.rb

```diff
@@ -0,0 +1,21 @@
+class Station
+  attr_reader :name,
+              :street_address,
+              :city,
+              :state,
+              :zip,
+              :full_address,
+              :fuel_type,
+              :access_time
+
+  def initialize(station_data)
+    @name = station_data[:station_name]
+    @street_address = station_data[:street_address]
+    @city = station_data[:city]
+    @state = station_data[:state]
+    @zip = station_data[:zip]
+    @full_address = "#{@street_address} #{@city}, #{@state} #{@zip}"
+    @fuel_type = station_data[:fuel_type_code]
+    @access_time = station_data[:access_days_time]
```

Comment:
I'd consider making @ full_address into just an instance method for this class, OR, handle the formatting of the address in the view. I'm suggesting not to make it an instance variable with an attr_reader, since all of those attributes are already present. It's persisting data that is already persisted.